applications/IgaApplication/custom_utilities/node_curve_geometry_3d.h

```diff
@@ -93,37 +93,37 @@ class NodeCurveGeometry3D
 
     /** Gets the location of the Kratos node at a given index.
      * 
-     * \param Index Index of the node
+     * @param Index Index of the node
      * 
      * @return Location of the Kratos node at the given index.
      */
     VectorType Pole(
         const int Index) const override
     {
-        const auto& node = *Node(Index);
+        const Kratos::Node<3>& node = *Node(Index);
  
         VectorType pole;
-        pole[0] = node[0];
-        pole[1] = node[1];
-        pole[2] = node[2];
+        for (size_t i = 0; i < 3; i++) {
+            pole[i] = node[i];
+        }
  
         return pole;
     }
 
     /** Sets the location of the Kratos node at a given index.
      * 
-     * \param Index Index of the node
-     * \param Value New location of the Kratos node
+     * @param Index Index of the node
+     * @param Value New location of the Kratos node
      */
     void SetPole(
         const int Index,
         const VectorType& Value) override
     {
-        auto& node = *Node(Index);
- 
-        node[0] = Value[0];
-        node[1] = Value[1];
-        node[2] = Value[2];
+        Kratos::Node<3>& node = *Node(Index);
```

Comment:
Define typedef Node<3> NodeType;

Maybe in the future we remove the template from Node

Reply:
BTW: Does the current definition of the node template make sense? I think it always inherits from a Point<3> or something like that...

Reply:
Afaik also this template is gonna be removed